app/Console/Commands/ImportNasspGroupsCommand.php

```diff
@@ -86,44 +86,61 @@ public function handle()
             'name' => 'National Student Council',
             'filter_by_state' => true,
         ]);
+        $nasspGroupType = GroupType::firstOrCreate([
+            'name' => 'National Association of Secondary School Principals',
+            'filter_by_state' => true,
+        ]);
 
         info('rogue:nassp-groups-import: Beginning import...');
 
         /**
          * Without specifying these column names, this command fails with error:
          * "The header record must be an empty or a flat array with unique string value."
          */
-        $records = $csv->getRecords(['LABEL NAME', 'NHS', 'NJHS', 'NASC', 'NASSP', 'ADDRESS1', 'ADDRESS2', 'CITY', 'STATE', 'POSTAL', 'COUNTRY', 'NCES']);
+        $records = $csv->getRecords(['name', 'NHS', 'NJHS', 'NASC', 'NASSP', 'city', 'state', 'zip', 'universalid', 'ncescode', 'gsid']);
```

Comment:
Just curious if there's a reason for the column name variety in capitalization and why not the same across the different columns?

Reply:
The uppercase was leftover from the initial file used in #1056. I updated the new file to have all lowercase and made corresponding changes in https://github.com/DoSomething/rogue/pull/1059/commits/8789aef509c8412157c73616c509d27bc4dd7e23. Thanks for the thorough review!